docs/migration/4_0.md

```diff
@@ -426,6 +426,7 @@ OrderOverviewComponent uses different translation label keys:
 - Key `checkoutProgress.methodOfPayment` is replaced by `orderDetails.methodOfPayment`.
 - Key `checkoutPO.costCenter` is replaced by `orderDetails.costCenter`.
 - Key `checkoutShipping.shippingMethod` is replaced by `orderDetails.shippingMethod`.
+- The `getOrderCurrentDateCardContent` method requires a function parameter. It is no longer optional.
```

Comment:
It can be understood differently if we read function as a type. Actually it's a string.

I would say that this function now requires isoDate parameter (I would avoid "function" wording here).

Same applies to commment in migration